app/views/sufia/homepage/_recent_document.html.erb

```diff
@@ -1,18 +1,13 @@
     <tr>
-      <% if display_thumbs %>
-        <td class="col-sm-2">
-          <%= link_to_profile recent_document.depositor("no depositor value") %>
-          <%= link_to curation_concerns_generic_work_path(recent_document) do %>
-            <%= render_thumbnail_tag recent_document, { width: 45 }, false %>
-          <% end %>
-        </td>
-      <% end %>
+      <td class="col-sm-2">
+        <%= link_to_profile recent_document.depositor("no depositor value") %>
+        <%= link_to curation_concerns_generic_work_path(recent_document) do %>
+          <%= render_thumbnail_tag recent_document, { width: 45 }, false %>
+        <% end %>
+      </td>
       <td>
         <h3>
           <span class="sr-only">Title</span><%= link_to truncate(recent_document.title_or_label, length: 28, separator: ' '), curation_concerns_file_set_path(recent_document)  %>
-          <% if display_access %>
```

Comment:
It was put there way back when this partial was called from different places and we wanted it to show in one place and not another.  Now it's only called from one place, so we don't need the conditional.